geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/usage/UsageStatWriter.scala

```diff
@@ -25,9 +25,7 @@ import org.locationtech.geomesa.accumulo.util.GeoMesaBatchWriterConfig
 class UsageStatWriter(connector: Connector, table: String) extends Runnable with Closeable with LazyLogging {
 
   // initial schedule
-  UsageStatWriter.executor.schedule(this, writeDelayMillis, TimeUnit.MILLISECONDS)
-
-  private val writeDelayMillis = 5000
+  UsageStatWriter.executor.schedule(this, UsageStatWriter.writeDelayMillis, TimeUnit.MILLISECONDS)
```

Comment:
i think the problem was using the val writeDelayMillis before it is declared. although in that case you would think it would default to 0 and execute immediately...